aws_lambda_decorators/validators.py

```diff
@@ -3,6 +3,18 @@
 import re
 from schema import SchemaError
 
+CURRENCIES = {"AED", "AFN", "ALL", "AMD", "ANG", "AOA", "ARS", "AUD", "AWG", "AZN", "BAM", "BBD", "BDT", "BGN", "BHD",
```

Comment:
Are there currencies missing? CHE is one that I could find, but I have a list of 178 different currencies (which could be wrong) and there are not that many in this set.

Reply:
I'll double check

Reply:
I extracted the new one from the iso xls on their website, removing the duplicates there are 169 and some bonds and precious metal ones about 10 of which have been removed from our set.

Reply:
ok cool